mesop/components/table/table.py

```diff
@@ -38,19 +38,39 @@ def map_table_click_event(event, key):
 register_event_mapper(TableClickEvent, map_table_click_event)
 
 
+@dataclass(kw_only=True)
+class TableHeader:
+  sticky: bool = False
+
+
+@dataclass(kw_only=True)
+class TableColumn:
+  sticky: bool = False
+
+
 # Don't include type hint since Pydantic can't properly type check the Pandas data
 # frame. In addition, we don't want to include Pandas as a dependency into Mesop.
 @register_native_component
 def table(
-  data_frame: Any, *, on_click: Callable[[TableClickEvent], Any] | None = None
+  data_frame: Any,
+  *,
+  on_click: Callable[[TableClickEvent], Any] | None = None,
+  header: TableHeader | None = None,
+  columns: dict[str, TableColumn] | None = None,
 ):
   """
   This function creates a table from Pandas data frame
 
   Args:
       data_frame: Pandas data frame.
       on_click: Triggered when a table cell is clicked. The [click event](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/click_event) is a native browser event.
+      header: Configures table header to be sticky or not.
+      columns: Configures table columns to be sticky or not.
```

Comment:
I would add a comment on what the key for the dict should be (e.g. column name)

Reply:
Good call. Updated.